### PR TITLE
Add AzureCLI 2.38.0

### PR DESCRIPTION
**Pull request by azclibot**
Add AzureCLI 2.38.0

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65123)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
need to move to multi-file

**Comment by jedieaston** [comment]
@fengzhou-msft is it easy to convert this to a multifile manifest?

**Comment by fengzhou-msft** [comment]
> @fengzhou-msft is it easy to convert this to a multifile manifest? If you can use wingetcreate with the bot it does it, if not, we can help write a script to do it.

@jiasli @yonzhan to take care of this as I no longer work on Azure CLI.

File: manifests/m/Microsoft/AzureCLI/2.38.0/Microsoft.AzureCLI.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PackageName: Microsoft Azure CLI
+Publisher: Microsoft Corporation
+Moniker: azure-cli
+PackageUrl: https://docs.microsoft.com/en-us/cli/azure/
+License: Copyright (C) Microsoft Corporation
+ShortDescription: The Azure command-line interface (Azure CLI) is a set of commands used to create and manage Azure resources. The Azure CLI is available across Azure services and is designed to get you working quickly with Azure, with an emphasis on automation.
+Installers:
+- Architecture: x64
+  InstallerUrl: https://azcliprod.azureedge.net/msi/azure-cli-2.38.0.msi
+  InstallerType: msi
```
Comment (Trenly) [inline manifests/m/Microsoft/AzureCLI/2.38.0/Microsoft.AzureCLI.yaml:12]:
```suggestion
  InstallerType: wix
```

**Pull request closed without merging**